src/pretix/control/templates/pretixcontrol/event/payment.html

```diff
@@ -48,19 +48,19 @@ <h1>{% trans "Payment settings" %}</h1>
                                 </a>
                             </td>
                         </tr>
-                    {% empty %}
+                    {% endfor %}
                         <tr>
-                            <td colspan="3">
+                            <td colspan="4">
+                                <br>
                                 {% url "control:event.settings.plugins" event=request.event.slug organizer=request.organizer.slug as plugin_settings_url %}
-                                {% blocktrans trimmed with plugin_settings_href='href="'|add:plugin_settings_url|add:'"'|safe %}
-                                    There are no payment providers available. Please go to the
-                                    <a {{ plugin_settings_href }}>plugin settings</a> and activate one or more payment plugins.
-                                {% endblocktrans %}
+                                <a href="{{ plugin_settings_url }}#tab-0-1-open" class="btn btn-default">
+                                    <i class="fa fa-plus"></i> {% trans "Activate additional payment plugins" %}
```

Comment:
```suggestion
                                    <i class="fa fa-plus"></i> {% trans "Enable additional payment plugins" %}
```

for consistent verbiage